apps/dashboard/app/apps/ood_app.rb

```diff
@@ -72,9 +73,10 @@ def links
           new_tab: true
         )
       ].concat(
-        OodFilesApp.new.favorite_paths.map do |path|
+        OodFilesApp.new.favorite_paths.map do |fp|
```

Comment:
I understand the temptation and I may have used it in an example I shared with you, but it is preferable to expand "fp" to the full word "favorite_path" i.e.

```ruby
OodFilesApp.new.favorite_paths.map do |favorite_path|
  OodAppLink.new(
    title: favorite_path.title || favorite_path.path.to_s,
    # etc...
```